src/clj/nnts2/handler/note.clj

```diff
@@ -1,14 +1,18 @@
 (ns nnts2.handler.note
   (:require [ring.util.response :as res]
-            ;[clj-time.core :as time]
             [nnts2.db.note :as db]
             [nnts2.model.note :as spec]
             [nnts2.spec-helpers :as spec-helper]))
 
-#_(defn now [] (new java.util.Date))
 
 (defn create [{:keys [body nnts-user]}]
   (let [note-data (assoc body :created-by-id nnts-user)]
     (if (spec/valid? note-data)
-      (res/response (db/add note-data))
+      (res/response (db/create note-data))
       (spec-helper/invalid (spec/explain-str note-data)))))
+
+
+(defn get-notes [{:keys [nnts-user params]}]
+  (let [get-params (select-keys params [:id :title :content :created-at])
```

Comment:
`get-params` and `params` are similarly named and can be confusing. It's probably not worth naming `get-params` separately to begin with. You can thread `params` through `select-keys` and `assoc` and pass it to `db/get`.